simulation/simtypes/randutil.go

```diff
@@ -74,3 +76,17 @@ func (sim *SimCtx) RandSubsetCoins(coins sdk.Coins) sdk.Coins {
 	r := sim.GetSeededRand("random subset coins")
 	return sdkrand.RandSubsetCoins(r, coins)
 }
+
+// Random float point from 0-1
+func (sim *SimCtx) RandomWeight(maxVal sdk.Dec) (sdk.Dec, error) {
```

Comment:
Also, we have a `RandomDecAmount` function. Why cannot we use that instead?

Reply:
removed to use RandomDecAmount